docs/web/usecases-implement-component-with-ui.mdx

```diff
@@ -1,9 +1,11 @@
 ---
-title: Custom Component with UI
+title: Build a NewsFeedTicker Component
```

Comment:
Minor gripe, but personally I preferred the old title since it better captured the actual use case. Only a very tiny subset of customers would ever need to build a news feed ticker component :-). There is a distinction between "use cases" and "examples", and this makes it sound more like the latter.

Reply:
That's a great point. It's also the odd title out in the SDK use cases for that reason.

One change we wanted to make was make the use cases start with action words if possible, so I might change it to "Build a Custom Component with UI"

Reply:
Updated! resolving